src/Controls/tests/UITests/Tests/Issues/Issue21706.cs

```diff
@@ -0,0 +1,25 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.AppiumTests.Issues
+{
+	public class Issue21706 : _IssuesUITest
+	{
+		public override string Issue => "ImageButton stuck in PointerOver state";
+
+		public Issue21706(TestDevice device) : base(device) { }
+
+		[Test]
+		public async Task ImageButtonStuckAfterRightClick()
+		{
+			this.IgnoreIfPlatforms(new TestDevice[] { TestDevice.Android, TestDevice.Mac, TestDevice.iOS });
```

Comment:
Any specific reason we want to not test this on the other platforms?

I get that this is about clicks so it might not work on some platforms, but if it reasonably can work, why not let them run there as well?

Reply:
Ah, the ticket mentioned that it already worked fine in macOS, so I wrote the test to only check on Win. The Appium test does specifically use the pointer right click, so we could enable it for macOS as well.